### DISCUSSION TITLE
2023-01-12のミーティングアジェンダ

### DISCUSSION THREAD

**Discussion by azu** [asciidwango/js-primer#1544] [Meeting]
2023-01-12のミーティングアジェンダ置き場です。

**Comment by lacolaco** [comment]
前回はアジェンダだけでミーティング不実施だった 
- https://github.com/asciidwango/js-primer/discussions/1542

**Comment by azu** [comment]
https://github.com/nodejs/Release/issues/778#issuecomment-1265458751

1月18日

**Reply by azu** [reply]
Node 18にnpm9が入る修正のPRを出す

**Comment by azu** [comment]
https://github.com/asciidwango/js-primer/pulls?q=is%3Apr+is%3Aclosed+label%3A%22Type%3A+Need+to+Publish%22

**Comment by azu** [comment]
レビューアーをどうするか

- とりあえず募集してみる

**Comment by azu** [comment]
- [今年のオープンソース活動振り返り @ 2022 | Web Scratch](https://efcl.info/2022/12/31/open-source-in-2022/)
  - [📕 Railsガイド『協賛プラン』をリリース｜YassLab 株式会社｜note](https://note.com/yasslab/n/n760b01d350d7)

**Reply by azu** [reply]
https://www.oreilly.com/online-learning/teams-jp.html

**Reply by azu** [reply]
[Liberapay](https://liberapay.com/)

**Reply by azu** [reply]
Open Collectiveを作ってみる

**Comment by azu** [comment]
- [ ] Node.js 18にnpm 9の取り込む PR出す

**Reply by azu** [reply]
2/14 までに Node.js npm9backportが入ってなかったら入れない。

**Reply by azu** [reply]
npm 9でインストールが通るかはチェックする

**Comment by azu** [comment]
次回
2/16 19:30〜
